### DISCUSSION TITLE
Keyword Highlighting in Terminal

### DISCUSSION THREAD

**Discussion by moisestohias** [kovidgoyal/kitty#7623] [Ideas]
One of the features that I find really helpfull is generic RegEx Keyword syntax highlights for all text (similar to MobaXterm on Windows). This helpful to spot error and common patters. 
What is the chance of implementing?
I tried the built-in [marks](https://sw.kovidgoyal.net/kitty/marks/) feature, it's not the best solution.
Currently I am using [ChromaTerm](https://github.com/hSaria/ChromaTerm/) which works to fine to certain extent & [highlight](https://www.gnu.org/software/src-highlite/source-highlight.html)

**Comment by kovidgoyal** [comment]
Just change your scrollback pager to nvim or something else that supports whatever type of highlighting you like and press ctrl+shift+h
